dev/NavigationView/NavigationView.cpp

```diff
@@ -1358,6 +1362,36 @@ void NavigationView::UpdateIsClosedCompact()
     }
 }
 
+void NavigationView::UpdatePaneButtonsWidths()
+{
+    auto newButtonWidths = [this]()
+    {
+        if (DisplayMode() == winrt::NavigationViewDisplayMode::Minimal)
+        {
+            return static_cast<double>(c_paneToggleButtonWidth);
+        }
+        return CompactPaneLength();
+    }();
+
+    if (const auto backButton = m_backButton.get())
+    {
+        backButton.Width(newButtonWidths);
+    }
+    if (auto&& paneToggleButton = m_paneToggleButton.get())
+    {
+        paneToggleButton.MinWidth(newButtonWidths);
+        if (const auto&& iconGridColumnElement = paneToggleButton.GetTemplateChild(c_paneToggleButtonIconGridColumnName))
```

Comment:
>const auto&& [](start = 12, length = 12)

The difference between auto&& and auto& is the former perserves the original values const-ness. So if the right hand value is not const then the resulting local wont be const either.  therefore you should never combine const and auto&&.  const auto& makes sense but double & doesn't.

Reply:
Oh I definitely did not knew this. Thank you for the explanation!